azure-kusto-data/.eslintrc.js

```diff
@@ -0,0 +1,246 @@
+/*
```

Comment:
Is it possible to use single .eslintsrc to both projects?

Reply:
I couldn't make it work sadly, I even tried asking stackoverflow - https://stackoverflow.com/questions/71224192/share-eslintrc-js-between-sibling-directories

But no response..